src/components/HelpResources/HelpResources.js

```diff
@@ -1,12 +1,21 @@
 import React from "react";
+import { GridContainer, Grid } from "@trussworks/react-uswds";
+import { HelpResourcesCards } from "../HelpResourcesCards/HelpResourcesCards";
 
 export const HelpResources = () => {
   return (
-    <div>
-      <h1 style={{ textAlign: "left" }}>Help & Resources</h1>
-      <p style={{ textAlign: "left" }}>
-        What to find in Help & Resources sections
-      </p>
-    </div>
+    <GridContainer>
+      <Grid row gap>
+        <Grid desktop={{ col: 8 }} className="text-left">
+          <h1 style={{ textAlign: "left" }}>Help & Resources</h1>
```

Comment:
These textAligns shouldn't be necessary after merging in Karina's changes. We'll wait on those before merging, but you can remove them here.